upi/aws/cloudformation/03_cluster_security.yaml

```diff
@@ -154,6 +154,26 @@ Resources:
       ToPort: 9999
       IpProtocol: tcp
 
+  MasterIngressInternalUDP:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId: !GetAtt MasterSecurityGroup.GroupId
+      SourceSecurityGroupId: !GetAtt MasterSecurityGroup.GroupId
+      Description: Internal cluster communication
```

Comment:
Would be nice to be a bit more specific here.  [The docs][1] have:

> Host level services, including the node exporter on ports `9100`-`9101`.

and similar wording a bit earlier for TCP.  But this is not _new_ to this PR (it echos the TCP CloudFormation entries above), so while I'd like to improve the docs here, I don't think it is a blocker for this PR.

[1]: https://github.com/openshift/openshift-docs/blame/7fa3e3eecdc3c28750b83b074784a4b5fa79ab1e/modules/installation-network-user-infra.adoc#L78-L79